survey-runner-application/security_groups.tf

```diff
@@ -0,0 +1,86 @@
+# Our default security group to access
+# the instances over SSH and HTTP
+resource "aws_security_group" "survey_runner_default" {
+  name        = "${var.env}-survey-runner-application-sg"
+  description = "eQ application security group"
+  vpc_id      = "${var.vpc_id}"
+
+  # HTTP access from the VPC
+  ingress {
+    from_port   = 80
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = ["${var.vpc_cidr_block}"]
+  }
+
+  ## AMQP Connection ports
+  ingress {
+    from_port   = 5672
+    to_port     = 5672
+    protocol    = "tcp"
+    cidr_blocks = ["${var.vpc_cidr_block}"]
+  }
+
+  egress {
```

Comment:
I've removed all explicit ports as there is a rule that allows egress for everything (needed for the Elastic Beanstalk builds)